### PR TITLE
[release-1.1.0] Bump Contour to v1.19.1

### PR DESCRIPTION
**Pull request by carlisia**
Closes https://github.com/knative-sandbox/net-contour/issues/648.

Question: is `v1.0.1` the next version we want for net-contour?

### CODE REVIEW

**Comment by nak3** [comment]
> Question: is v1.0.1 the next version we want for net-contour?

No, the next version is v1.1 and I think you just need to send the PR against `main` branch as it was not cut yet.

File: go.mod [hunk 1]
```diff
@@ -3,18 +3,15 @@ module knative.dev/net-contour
...
 	github.com/google/go-cmp v0.5.6
 	github.com/mailru/easyjson v0.7.7 // indirect
 	github.com/mikefarah/yq/v3 v3.0.0-20200601230220-721dd57ed41b
-	github.com/projectcontour/contour v1.18.2
+	github.com/projectcontour/contour v1.19.1
 	go.uber.org/zap v1.19.1
-	k8s.io/api v0.21.4
-	k8s.io/apimachinery v0.21.4
-	k8s.io/client-go v0.21.4
+	k8s.io/api v0.22.0
```
Comment (nak3) [inline go.mod:12]: Contour 1.19 requires k8s 1.22 deps but Knative still needs to use 1.21 deps until https://github.com/knative/pkg/pull/2327 is in. So we need to pin k8s libs in the `replace` field below like this:

```
replace (
        github.com/envoyproxy/go-control-plane => github.com/envoyproxy/go-control-plane v0.9.10-0.20210614203518-782de910ff04
        k8s.io/api => k8s.io/api v0.21.4
        k8s.io/apimachinery => k8s.io/apimachinery v0.21.4
        k8s.io/client-go => k8s.io/client-go v0.21.4
)
```

Comment (carlisia) [reply go.mod:12]: Ah thanks for this! Will change it.

**Comment by carlisia** [comment]
Gonna try to fix this by opening a new PR.

**Comment by carlisia** [comment]
See: https://github.com/knative-sandbox/net-contour/pull/689#issuecomment-993863919.

**Pull request closed without merging**